crates/forge-runner/src/test_case_summary.rs

```diff
@@ -159,8 +163,9 @@ impl TestCaseSummary<Fuzzing> {
                 TestCaseSummary::Passed {
                     name,
                     msg,
-                    gas_info: GasStatistics::new(&gas_usages),
                     arguments,
+                    gas_info: GasStatistics::new(&gas_usages),
+                    used_resources: UsedResources::default(),
```

Comment:
Shouldn't we at least calculate means of used resources?

Reply:
Let's do it in another PR

Reply:
@drknzz would you submit an issue?

Reply:
Yes, will submit all of the further issues from this pr before merge

Reply:
#1628